cloud-data-protection/src/services/authService.tsx

```diff
@@ -0,0 +1,23 @@
+import store from '../stores/Store';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
import store from "../stores/Store";
```